examples/network/nodeStyles/circularImagesWithPosition.html

```diff
@@ -0,0 +1,109 @@
+<!doctype html>
+<html>
+<head>
+  <title>Vis Network | Node Styles | Circular Images</title>
```

Comment:
Could you consolidate those two examples into a single one? Also change the title (that's the thing that actually appears in examples index).

Reply:
I agree with the title change. But won't it get too messy to have both examples in the same file?  I'm just following what's already done, and there seems to be 1 file for each image and circular image case.

Reply:
It's a single example, that's why I want you to merge it. You are showcasing single thing: positioning. It works with two node shapes so just make some nodes image and some circular image.